config/routes.rb

```diff
@@ -8,9 +8,20 @@
   # config/routes.rb
   root 'users#index'
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show new create] do
-      resources :comments, only: %i[new create]
+    resources :posts, only: %i[index show new create destroy] do
+      resources :comments, only: %i[new create destroy]
       resources :likes, only: %i[new create]
     end
   end
+
+  namespace :api do
+    resources :users, only: %i[none] do
+      resources :posts, only: %i[index]
+    end
+    
+   
+    resources :posts, only: %i[none] do
+      resources :comments, only: %i[index new create]
+    end
+  end
 end
```

Comment:
- [ ] Hi @ctokoli I can see you identify you URL as an API by using `/api` :+1: but it seems controllers didn't follow this same `/api` pattern. When I sent a GET request to get all posts for a user(`http://localhost:3000/api/users/1/posts`), I didn't get a correct response in JSON format. Please kindly make sure you fix this issue and make sure all API endpoints receive JSON and response in JSON format as well. We should be able to get all posts for a user, get all comments for a post and add comments to a post in JSON format.

Reply:
Let's kindly make sure both team members contribute actively in this project through commit messages since this is a collaborative project. @rabayarabu committed only once.